app/src/interfaces/translations/translations.vue

```diff
@@ -244,6 +247,9 @@ function useLanguages() {
 
 		fields.add(pkField);
 
+		// Open for discussion, since this field might need to be dynamic
+		fields.add('direction');
```

Comment:
You guessed it! This has to be dynamic. The more difficult question is where that dynamic configuration lives. I think it'll have to be on the relationship that defines the translation, just like we're defining the language code / name 🤔

Reply:
Would it be an idea to add this as a configuration to the translation interface options? Just like there's an option to change the Language Indicator Field.

![image](https://user-images.githubusercontent.com/73393707/181211023-96746ea3-c124-4f4a-8411-8dd1f75fd1c4.png)

A _Language Direction Field_ for example. 

I think it's a good idea to add the direction field by default to the Languages collection creation, like we already did in the PR. Since this is probably the most used way to configure a multilanguage setup in Directus. Therefore we could set the _Language Direction Field_ default to this direction field. Where it still stays configurable. 

Happy to hear your thoughts!

Reply:
Yes, that'd be the perfect place 👍🏻 It's only the `translations` interface that "cares" about what field is what in this context, so direction could be a part that. 

> I think it's a good idea to add the direction field by default to the Languages collection creation, like we already did in the PR. Since this is probably the most used way to configure a multilanguage setup in Directus. Therefore we could set the Language Direction Field default to this direction field. Where it still stays configurable.

Makes sense to me!

Reply:
Cool, I've worked it out as described in the last two commits. The translation interface includes an extra field called languageDirectionField. When the language collection has a direction field it becomes the default value.

Next to that I updated the default language collection creation. The direction field had a dropdown interface with two choices. Left to Right and Right to Left.

I added all nessesary labels/translations to the en-US.yaml. Is this enough or do I need to add the labels to all the other language yaml's as well?